### PR TITLE
Adding in the left-side-bar, fit it to the whole side of the page so …

### PR DESCRIPTION
**Pull request by dortega5185**
…it will follow user. Also commented out filter system to be moved to side-bar in next feature update

## Description ##

Clearly documents the reason for the pull request and what has changed.
- [ ] This is a bugfix.
- [x] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [x] Have you linked this pull request to a Trello card?

## Review ##

- [x] Have you added two reviewers to this pull request?
- [ ] Is this a work in process?
- [x] Is this ready to be reviewed?

### CODE REVIEW

**Review by domikitchen — COMMENTED** [review]
awesome job

**Review by domikitchen — COMMENTED** [review]
Looks great (last review went in as a comment, meant to approve)

**Review by domikitchen — APPROVED** [review]

File: src/components/pages/DataViz/Graphs/LeftSideBar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import * as AiIcons from 'react-icons/ai';
+import { Link } from 'react-router-dom';
+import '../../../../styles/sass/left-side-menu.scss';
+import { IconContext } from 'react-icons';
+import Logo from '../../../../styles/imgs/B2P_Symbol_Green.png';
+
+function LeftSideBar(props) {
+  return (
+    <div className="bigBoyContainer">
```
Comment (Vippsi) [inline src/components/pages/DataViz/Graphs/LeftSideBar.js:10]: Love the className

**Review by Vippsi — APPROVED** [review]
Everything looks great. Good work!

File: src/components/pages/DataViz/Graphs/LeftSideBar.js (hunk 1 above)
Comment (dortega5185) [reply src/components/pages/DataViz/Graphs/LeftSideBar.js:10]: Hell yeah!

**Pull request merged**